extensions/amp-story/1.0/animation.js

```diff
@@ -192,9 +204,20 @@ export class AnimationRunner {
    */
   getDims() {
     return this.vsync_.measurePromise(() => {
-      const targetRect = unscaledClientRect(this.target_);
+      const target = dev().assertElement(this.presetTarget_);
+      const targetRect = unscaledClientRect(target);
       const pageRect = unscaledClientRect(this.page_);
 
+      // TODO(alanorozco): Expose equivalents to <amp-animation>
+      // - targetWidth/targetHeight are already available as width()/height()
+      // - pageWidth/pageHeight should be exposed as vw/vh
+      // - targetX/targetY should be exposed somehow (?)
+      //
+      // TODO(alanorozco): After exposing these to <amp-animation> syntax, we
+      // can get rid of this entire method (and this async chain!) if we ensure
+      // that presets avoid visual jumps either via:
+      // a) default styles and/or
+      // b) by not using special <amp-animation> syntax in initial keyframe.
```

Comment:
Can you track all this with an issue? This is a lot of context and it looks like it deserves proper tracking / context sharing. :))